processing/src/main/java/io/druid/query/topn/TopNMapFn.java

```diff
@@ -33,6 +33,10 @@
 
 public class TopNMapFn
 {
+  public static final long ZERO_LONG = 0L;
```

Comment:
will make is as an object right now and will stop there, i think this will be changed soon with the upcoming null support soon.